tests/test_adminpage.py

```diff
@@ -1,40 +1,45 @@
+from cms.api import create_page
 from cms.models import Page
+from cms.test_utils.testcases import CMSTestCase
 from django.contrib import admin
 
-from . import BaseTest
-
 page_admin = admin.site._registry[Page]
 
 
-class AdminPageTest(BaseTest):
+class AdminPageTest(CMSTestCase):
     def test_get_form_no_obj(self):
         """
         Test that the returned form has not been modified by the meta patch
         when no page object is specified
         """
-        request = self.get_page_request(None, self.user, "/", edit=True)
+        request = self.get_page_request(None, self.get_superuser(), "/")
```

Comment:
self.user should be ok here:

```suggestion
        request = self.get_page_request(None, self.user, "/")
```